src/index.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import App from './components/App';
+import App from './components/App/App';
```

Comment:
For readability, in `./components/App/` folder, we could create file `index.js` with content:
 ```js
 export { default } from './App';
 ```
By doing this 
```js
import App from './components/App/App';
```
becomes
 
```js
import App from './components/App';
```